### DISCUSSION TITLE
Converting user account to organisation... doesn't work

### DISCUSSION THREAD

**Discussion by JDiSolutions** [community/community#51304] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I am trying to convert a user account into an organisation. I have a user to transfer ownership to and have added that and completed the form, got past the warnings etc. It says I'll be redirected when it's done, and I am redirected but all that's happened is that the apps have been uninstalled. I am still logged in as the user I wanted to convert and no organisation has been created.

I've tried this a few times now, each time with the same result. Any ideas?

Cheers

**Comment by Anmol-Baranwal** [comment]
It sounds like the conversion process may have encountered an error. I recommend reaching out to GitHub support for assistance with this issue.

You can contact GitHub support by visiting their support page (https://support.github.com/contact?tags=docs-generic) and selecting the appropriate category for your issue. From there, you can submit a ticket describing the problem you're experiencing with converting your user account to an organization. Be sure to provide as much detail as possible, including any error messages you may have received.

GitHub support should be able to assist you in resolving the issue and completing the conversion process.

**Reply by Anmol-Baranwal** [reply]
@JDiSolutions 
I wanted to follow up and ask if you have any additional questions or concerns about my previous response on GitHub. If you have no further inquiries, would you kindly consider marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by JDiSolutions** [reply]
> @JDiSolutions I wanted to follow up and ask if you have any additional questions or concerns about my previous response on GitHub. If you have no further inquiries, would you kindly consider marking my answer as accepted?

Thanks, but we already submitted a support request before opening this issue (which has now been open for almost a week.) We were very much hoping that somebody might have some suggestions that don't involve support since this issue is proving disruptive to us and support doesn't seem to be responding in a timely manner.

**Comment by specky-rum** [comment]
Turns out we had some "classic" projects but these were all auto created by Waffle many years ago, and since Waffle no longer exists it was safe to delete them all. After doing that, the convert to organisation process worked.

**Comment by specky-rum** [comment]
Unfortunately I can't accept [that](https://github.com/orgs/community/discussions/51304#discussioncomment-5578712) as the answer because the account I used to create it is now an organisation so if anyone can accept that answer on our behalf, please do so.
